rust-keybroker/Cargo.toml

```diff
@@ -26,4 +26,4 @@ sha2 = "0.10.8"
 stderrlog = "0.6.0"
 thiserror = "1.0.63"
 tsm_report = { git = "https://github.com/veracruz-project/cca-utils-rs.git", rev = "cb88b76da722f2991365b159e3d575249dfbbe7d"}
-veraison-apiclient = { git = "https://github.com/veraison/rust-apiclient.git" }
+veraison-apiclient = { git = "https://github.com/veraison/rust-apiclient.git", tag = "c-bindings"}
```

Comment:
Curious as to why we're pinning to the `c-bindings` tag? The key broker doesn't require the C bindings, and indeed we'll need a newer tag than that if we want to embrace the HTTPs support.

Reply:
Because this is the only tag we have (and luckily it does correspond to the commit we are interested in). The tag name could be "more standard" (e.g. v0.0.Y which happens to be the last version with c-binding).
Cargo allows to pin to a specific version using a git tag, a git sha1 (not very friendly) or a Cargo version (which means the version field in the dependency's Cargo.toml  needs proper attention when we are updating stuff in the dependency).
This weird-looking line will disappear with my next PR (adding TLS support) because it will use version = "0.0.2" as the veraison-apiclient is properly versioned then.

Reply:
Understood. No problem if this is just a transient state of affairs.